src/app/auth/authentication.service.ts

```diff
@@ -138,6 +146,7 @@ export class AuthenticationService {
       let options = new RequestOptions({ headers: headers });
       return this.http.get(tokenUrl, options)
         .map(response => processToken(response))
+        .do(token => localStorage.setItem(broker + '_token', token.access_token))
```

Comment:
@dgutride does this actually work? I never got anything put into local storage...